gslib/tests/test_rsync.py

```diff
@@ -2172,6 +2172,8 @@ def _Check1():
     _Check1()
 
   @unittest.skipUnless(UsingCrcmodExtension(), 'Test requires fast crcmod.')
+  @SkipForS3('The boto lib used for S3 does not handle objects '
+             'starting with slashes if we use V4 signature')
```

Comment:
This is a super nit, but don't spaces between words normally go after line breaks?
'objects'
' starting'

Also, please add a period.

Reply:
I think the space is required because Python will just concat the two strings
```
>>> s = ('some string'
...      'value')
>>> s
'some stringvalue'
```

Reply:
ah, nevermind, I think  you meant move the space to the next line

Reply:
Done